crates/core/c8y_api/Cargo.toml

```diff
@@ -11,7 +11,7 @@ repository = { workspace = true }
 [dependencies]
 async-trait = "0.1"
 clock = { path = "../../common/clock" }
-csv = "1.1"
+csv = "1.2.1"
```

Comment:
Yes, but in this case it has to be 1.2.1 because the method I used got added in 1.2.1 and just leaving `csv = "1.2"` means that depending on the lockfile,1.2.0 could be used resulting in compilation errors.